.github/workflows/githubActions.yml

```diff
@@ -0,0 +1,46 @@
+name: Github actions
+
+on: ["push", "pull_request"]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
```

Comment:
why choose 12 version choose at least 14/16

Reply:
got  it